test/unit/int_alma/scripts/helpers/almaEligibilityHelperTest.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+// almaEligibilityHelper.js unit tests
+
+var assert = require('chai').assert;
+var expect = require('chai').expect;
+var almaEligibilityHelperMocks = require('../../../../mocks/helpers/almaEligibilityHelperMocks').almaEligibilityHelperMocks;
+var basketMock = require('../../../../mocks/dw/order/BasketMgr');
+
+var baseReturn =
+    {
+        purchase_amount: 25000,
+        queries: [],
+        locale: 'fr_FR',
+        billing_address: {
+            title: 'address.jobTitle',
+            first_name: 'address.lastName',
+            last_name: 'address.firstName',
+            company: 'address.companyName',
+            line1: 'address.address1',
+            line2: 'address.address2',
+            postal_code: 'address.postalCode',
+            city: 'address.city',
+            country: 'address.countryCode.value',
+            state_province: 'address.stateCode',
+            phone: 'address.phone'
+        },
+        shipping_address: {
+            title: 'address.jobTitle',
+            first_name: 'address.lastName',
+            last_name: 'address.firstName',
+            company: 'address.companyName',
+            line1: 'address.address1',
+            line2: 'address.address2',
+            postal_code: 'address.postalCode',
+            city: 'address.city',
+            country: 'address.countryCode.value',
+            state_province: 'address.stateCode',
+            phone: 'address.phone'
+        }
+    };
+
+var baseBasket = basketMock.getCurrentBasket();
```

Comment:
What is the supported version of Ecmascript in this context?
I mean, could we replace `var` with `let`?